src/shared/components/create-case-filters/create-case-filters.component.html

```diff
@@ -23,5 +23,10 @@
     </select>
   </div>
 
-  <button type="submit" class="button" [disabled]="!isCreatable()">{{startButtonText}}</button>
+  <a (click)="apply()" (keydown.enter)="apply()" [class.disabled]="!isCreatable()" tabindex="{{isCreatable() ? 0 : -1}}" role="button" draggable="false" class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-8 govuk-button--start" data-module="govuk-button">
```

Comment:
We've got bindings against functions here. It was agreed in the last tech forum that we should avoid this wherever possible. Can you please refactor the `[class.disabled]` binding so it's bound to a property, not a function.